server/src/main/java/io/spine/server/aggregate/AggregateRepository.java

```diff
@@ -406,8 +406,13 @@ protected void setSnapshotTrigger(int snapshotTrigger) {
         this.snapshotTrigger = snapshotTrigger;
     }
 
-    @VisibleForTesting
-    public AggregateStorage<I> aggregateStorage() {
+    /**
+     * Returns the storage assigned to this aggregate.
+     *
+     * @return storage instance
+     * @throws IllegalStateException if the storage is null
+     */
+    protected AggregateStorage<I> aggregateStorage() {
```

Comment:
This method is used outside of tests already. Although it doesn’t need to be publicly visible in the same manner as `ProjectionRepository#projectionStorage()` and `RecordBasedRepository#recordStorage()` are protected.

Reply:
Good to see.